src/client/datascience/commands/activeEditorContext.ts

```diff
@@ -103,8 +106,21 @@ export class ActiveEditorContextService implements IExtensionSingleActivationSer
         if (this.docManager.activeTextEditor?.document.languageId === PYTHON_LANGUAGE) {
             this.onDidChangeActiveTextEditor(this.docManager.activeTextEditor);
         }
-        this.vscNotebook.onDidChangeNotebookEditorSelection(this.updateNativeNotebookContext, this, this.disposables);
-        this.vscNotebook.onDidCloseNotebookDocument(this.updateNativeNotebookContext, this, this.disposables);
+        this.vscNotebook.onDidChangeNotebookEditorSelection(
+            this.updateNativeNotebookInteractiveWindowOpenContext,
+            this,
+            this.disposables
+        );
+        this.vscNotebook.onDidOpenNotebookDocument(
```

Comment:
I added in this case for onDidOpen. I believe this was usually already covered by the onDidChangeActiveNotebook and onDidChangeActiveInteractiveWindow (as the new window is usually picked up as it becomes active). But it felt safer to also have this onDidOpen in case there is a way to go open without becoming the active doc.